src/safeds/ml/classical/classification/_classifier.py

```diff
@@ -98,3 +99,39 @@ def accuracy(self, validation_or_test_set: TaggedTable) -> float:
         predicted = self.predict(validation_or_test_set.features).target
 
         return sk_accuracy_score(expected._data, predicted._data)
+
+    def precision(self, validation_or_test_set: TaggedTable, positive_class=1) -> float:
+        """
+        Compute the classifier's precision of the on the given data.
+
+        Parameters
+        ----------
+        validation_or_test_set : TaggedTable
+            The validation or test set.
+        positive_class : int | str
+            The classification to be considered positive
+
+        Returns
+        -------
+        precision : float
+            The calculated precision score, i.e. the ratio of correctly predicted positives to all predicted positives.
+            Returns 1 if no predictions are made.
+        """
+        expected = validation_or_test_set.target
+        predicted = self.predict(validation_or_test_set.features).target
+
+        if len(expected) != len(predicted):
+            raise AssertionError("Different length of 'expected' and 'predicted' vectors.")
```

Comment:
Not covered in test. Use a `ColumnLengthMismatchError` instead of an `AssertionError`.